src/features/card/utils/formattedString.ts

```diff
@@ -0,0 +1,37 @@
+import { DISPLAY_SECURITY_CHARACTER } from '@/features/card/constants/display';
+
+const formattedPaddedValue = (value: string, length: number) => value.padEnd(length, ' ');
+const formattedPasswordStyle = (value: string) => value.replace(/./g, DISPLAY_SECURITY_CHARACTER);
+
+export const formattedDisplayCardNumber = (cardNumber: {
+  first: string;
+  second: string;
+  third: string;
+  fourth: string;
+}) => {
+  if (!cardNumber.fourth && !cardNumber.third && !cardNumber.second) {
+    return cardNumber.first;
+  }
```

Comment:
이렇게 나열된 조건문은 이해하기 어렵고, 어떤 의도에서 작성된 건지 쉽게 파악할 수 없답니다.
조건문을 그룹화하여 지역 함수로 분리해보면 어떨까요?

```suggestion
const isFirst = !cardNumber.fourth && !cardNumber.third && !cardNumber.second;
if(isFirst) {
  return cardNumber.first;
}
```

Reply:
조건들에 변수화하여 처리해보도록 하겠습니다!